tests/select-input-test.js

```diff
@@ -0,0 +1,335 @@
+import { describe, it } from 'mocha'
+import React from 'react'
+import * as assert from 'assert'
+import Enzyme from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+import sinon from 'sinon'
+
+Enzyme.configure({ adapter: new Adapter() })
+
+import SelectInput from '../src/inputs/select-input.jsx'
+
+const exampleFormData = {
+  'select-typeahead': { value: '' },
+  'select-button': { value: '' }
+}
+const options = [
+  {
+    label: 'A',
+    value: 'a'
+  },
+  {
+    label: 'B',
+    value: 'b'
+  },
+  {
+    label: 'C',
+    value: 'c'
+  }
+]
+
+let value = 'c'
+
+const updateValue = (formId, fieldName, fieldValue) => {
+  value = fieldValue
+}
+
+describe('<SelectInput />', function() {
```

Comment:
If you'd like, you could also use arrow functions throughout the tests.  I don't think you will ever refer to `this` inside any of these blocks.